picard/file.py

```diff
@@ -158,6 +159,42 @@ def _load(self, filename):
         """Load metadata from the file."""
         raise NotImplementedError
 
+    def _load_preserved_config(self, metadata):
+        """Preserve current tag related config to metadata and returns True.
+        Returns False if no related configs"""
+        raise False
+
+    def _test_same(self):
```

Comment:
Can we call this _is_metadata_same  or (with opposite return values) _has_metadata_changed?